scripts/suse/apt/aptnotify

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
```

Comment:
Just wondering: is this python3 shebang causing any pain to a Python 2, in for example Ubuntu 16.04?